test/render/test-cases.js

```diff
@@ -791,35 +791,36 @@ export const TEST_CASES = [
         updateTriggers: {
           getIcon: 2
         },
-        sizeScale: 12,
+        sizeScale: 24,
         getPosition: d => d.COORDINATES,
         getColor: d => [64, 64, 72],
+        getSize: d => (d.RACKS > 2 ? 2 : 1),
         getIcon: d => {
           if (d.PLACEMENT === 'SW') {
             return {
-              url: './test/render/golden-images/pointcloud-meter.png',
+              url: './test/render/golden-images/examples/website_screen-grid.png',
```

Comment:
Can we not use a golden image as icon? They may change in the future for reasons unrelated to this layer.